go.mod

```diff
@@ -21,7 +21,7 @@ require (
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
 	github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e // indirect
 	github.com/golang/protobuf v1.3.5 // indirect
-	github.com/google/go-cmp v0.5.1 // indirect
```

Comment:
nit: this should be committed with the code which adds new import, so 3af34fae096718bd628f03ed41faafcad93dc5b9 in this case.